src/selectors/app.js

```diff
@@ -338,3 +340,9 @@ export const getTimelineMarginLeft: Selector<number> = createSelector(
     }
   }
 );
+
+export const getL10nState: Selector<L10nState> = state => state.l10n;
+export const getIsL10nFetching: Selector<boolean> = state =>
+  getL10nState(state).isL10nFetching;
+export const getLocalization: Selector<Localization> = state =>
+  getL10nState(state).localization;
```

Comment:
I guess I forgot to point out this earlier. Can you move these selectors to `src/selectors/l10n.js`? It's good practice to use same file name for actions, selectors and reducers, so we can easily switch between these.
Also a one line comment explaining what they are would be good.